community/clsync/APKBUILD

```diff
@@ -26,17 +26,35 @@ prepare() {
 build() {
 	cd "$builddir"
 
-	./configure \
-		CFLAGS="$CFLAGS -lfts -lexecinfo" \
-		--build=$CBUILD \
-		--host=$CHOST \
-		--prefix=/usr \
-		--sysconfdir=/etc \
-		--mandir=/usr/share/man \
-		--localstatedir=/var \
-		--enable-capabilities \
-		--enable-seccomp \
-		--disable-debug
+	case "$CARCH" in
+	aarch64)
+		./configure \
+			CFLAGS="$CFLAGS -lfts -lexecinfo" \
+			--build=$CBUILD \
+			--host=$CHOST \
+			--prefix=/usr \
+			--sysconfdir=/etc \
+			--mandir=/usr/share/man \
+			--localstatedir=/var \
+			--enable-capabilities \
+			--disable-seccomp \
+			--disable-debug
+		;;
+	*)
+		./configure \
+			CFLAGS="$CFLAGS -lfts -lexecinfo" \
+			--build=$CBUILD \
+			--host=$CHOST \
+			--prefix=/usr \
+			--sysconfdir=/etc \
+			--mandir=/usr/share/man \
+			--localstatedir=/var \
+			--enable-capabilities \
+			--enable-seccomp \
```

Comment:
I think it is beter we make a _seccomp variable that holds the value that is required.

Reply:
As you suggested, I have added a seccomp variable.

Please, kindly review.